.github/workflows/ci.yml

```diff
@@ -67,9 +67,17 @@ jobs:
 
     - name: Verify Pacts
       run: |
+        export IS_MERGED=${{ github.event.pull_request.merged }}
         export PACT_BROKER_BASE_URL='https://edx.pactflow.io'
         export PACT_BROKER_TOKEN=${{ secrets.PACT_FLOW_ACCESS_TOKEN }}
         export PUBLISH_VERSION=`git rev-parse --short HEAD`
-        export PUBLISH_TAGS=${{ github.head_ref }}
+        if [ $IS_MERGED == false ]
+        then
+          export PUBLISH_TAGS=${{ github.head_ref }}
```

Comment:
I believe we also have `GITHUB_HEAD_REF`, can we use it for sake of consistency?